src/translations/English.json

```diff
@@ -911,5 +912,12 @@
    "Choose Geyser split per transaction":"Choose Geyser split per transaction",
    "Lightning":"Lightning",
    "Onchain":"Onchain"
-
+   "Check out my project in geyser: {{projectLink}}":"Check out my project in geyser: {{projectLink}}",
```

Comment:
![Screenshot 2024-04-12 at 9 25 51 AM](https://github.com/geyserfund/geyser-app/assets/26449299/75fa3b78-2624-42aa-86fd-1fa5ac8b5344)

It shows this is the reason build it failing, but I do see a comma, make sure the build passes.